fulp_modules/features/species/beefmen/mob/beefman.dm

```diff
@@ -231,6 +232,23 @@
 		return FALSE
 
 	// Pry it off...
+	if(user.zone_selected == BODY_ZONE_PRECISE_MOUTH)
+		var/mob/living/carbon/C = user
+		var/obj/item/organ/tongue/beefman/tongue
+		for(var/org in C.internal_organs)
+			if(istype(org, /obj/item/organ/tongue/beefman))
```

Comment:
why are we only checking for beefmen tongues? shouldn't they be able to rip any tongue out?

Reply:
I thought being able to rip out and insert any type of tongue would be OP but i can change that